docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -753,6 +754,9 @@ A number of options are available.
 
 ```ts
 const yjsDocument = await liveblocks.getYjsDocument("my-room-id", {
+  // Optional, return a subdocument instead. guid is its unique identifier
+  guid: "c4a755...",
+
   // Optional, if true, `yText` values will return formatting
   formatting: true,
```

Comment:
We have not yet implemented `guid` support for the `getYjsDocument` node method. There has been discussion about deprecating this API overall especially because the result of this endpoint is not always consistent due to the lack of proper ways to decipher Yjs types on server. Do you reckon we should also add support for this endpoint?

Reply:
My mistake! Nah, I'll remove the documentation 🙂